### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by KarlGusta** [alefragnani/vscode-project-manager#610] [Q&A] [answered]
I would like to contribute to this project. Being a new member of this project, how can I run the project locally? Thanks.

**Comment by alefragnani** [comment, marked as answer]
Hi @KarlGusta ,

The extension is not fully open source anymore, so you can't run/build it locally.

I'm refactoring the `core` source and plans to reopen pieces of it, so anyone would be able to build/run the extension. Still no ETA.

Hope this helps

**Reply by KarlGusta** [reply]
Ooh. Thanks for the answer Alef. I won't stop complimenting this great product you have built. Thanks. Take care.

**Reply by luizgununes** [reply]
Hey @alefragnani, still without plans to make this project open source again? I use the extension everyday and would like to fix something that bothers me a little.
